src/pages/SearchPage/SearchPageFooter.tsx

```diff
@@ -2,19 +2,25 @@ import React, {useState} from 'react';
 import {View} from 'react-native';
 import ReferralProgramCTA from '@components/ReferralProgramCTA';
 import useThemeStyles from '@hooks/useThemeStyles';
+import * as User from '@userActions/User';
 import CONST from '@src/CONST';
 
 function SearchPageFooter() {
```

Comment:
This is confusing, but yes... this change is mostly redundant because the parent component is being removed in another PR. This was discussed over [here](https://github.com/Expensify/App/pull/33925#discussion_r1462460117).